sdk/identity/identity/samples/AzureCacheForRedis/ioredis.md

```diff
@@ -197,7 +204,11 @@ async function main() {
 
   async function updateToken() {
     accessTokenCache = await returnPassword(credential);
-    id = setTimeout(updateToken, ((accessTokenCache.expiresOnTimestamp- 120*1000)) - Date.now());
+    let randomTimestamp = randomNumber(120000,300000);
+    id = setTimeout(updateToken, ((accessTokenCache.expiresOnTimestamp- randomTimestamp)) - Date.now());
+    if(redis){
+      await redis.auth(accessTokenCache.token);
```

Comment:
Do we not need to pass "username" on this API ?

Reply:
For io-redis it was not needed for the auth command

Reply:
Just for confirmation, if you dont specify username, still this command will send username from previous contexts along with this new token? Is this code tested? otherwise auth will fail after this based on randomNumber(120000,300000), connection will be closed.

Reply:
I see node-redis.md file has username in argument.

Reply:
```suggestion
      await redis.auth(process.env.REDIS_SERVICE_PRINCIPAL_NAME, accessTokenCache.token);
```